database/postgresSeed.js

```diff
@@ -4,32 +4,49 @@ const randomNumberUpTo = (limit) => Math.floor(Math.random() * limit);
 
 // create listings and insert in batches of 10,000
 const seedListings = async () => {
-  for (let i = 0; i < 1000; i++) {
+  let t1 = Date.now();
+  for (let i = 0; i < 20000; i++) {
     let listingsBatch = [];
-    for (let j = 0; j < 10000; j++) {
+    let bookedDatesBatch = [];
+    let batchSize = 500;
```

Comment:
Has this been the most optimal batchSize for you? Make sure that your memory is around 1GB when running this script as we're only going to have that much with deployment

Reply:
Good question. I tested various batch sizes with Mongo but not yet for Postgres. I will do so.